ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesRelations/RelationSequence/RelationSequence.js

```diff
@@ -26,62 +33,59 @@ export default class RelationSequence extends Component {
         <RelationRemover
           referrer={seriesDetails}
           related={row}
+          relationType={row.relation_type}
           buttonContent={'Remove relation'}
         />
       );
     }
   };
 
-  languagesFormatter = ({ row }) => {
-    return <SeriesLanguages metadata={row} />;
-  };
-
   render() {
     const { relations, showMaxRows, isLoading, error } = this.props;
-    const next = relations['next'] || [];
-    const previous = relations['previous'] || [];
+
+    const sequenceRelations = _get(relations, 'sequence', []);
+    const continuations = sequenceRelations.filter(
+      rel => rel.relation_order === 'is_continued_by'
```

Comment:
does it make sense for `is_continued_by` and `continuous` to be part of the config?

Reply:
It might be... but it is not configurable, so it might be misleading to have it there.